src/Test/OpenRiaservices.EndToEnd.Wcf.Test/Data/DataServiceTests.cs

```diff
@@ -486,6 +486,9 @@ public void TestInvalidProviderName()
         /// Verify that if an invalid DomainOperationEntry name is specified, that the Load
         /// operation finishes with the expected WebResponse.StatusCode.
         /// </summary>
+#if ASPNETCORE
+        [Ignore("Does not work the same way with AspNetCore")]
```

Comment:
```suggestion
        [Ignore("BinaryHttpDomainClientFactory does not validate if method exists, and since name is always specified by code generation it is not important to validate it")]
```